libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1494,12 +1538,11 @@ public static String replaceMediaFileWithUrl(Context context, @NonNull String po
             addDefaultSizeClassIfMissing(attrs);
 
             // clear overlay
-            content.clearOverlays(predicate);
-            content.updateElementAttributes(predicate, attrs);
-            content.refreshText();
+            // content.clearOverlays(predicate);
```

Comment:
Is this no longer necessary? If there's a reason to leave it in, commented out, it could use an explanatory comment.